app/SmsStatus.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Chompy;
+
+class SmsStatus
+{
+    /**
+     * @var string
+     */
+    public static $active = 'active';
```

Comment:
These should probably be [class constants](https://www.php.net/manual/en/language.oop5.constants.php).

Reply:
This was following the example of our `ImportType` class, introduced per https://github.com/DoSomething/chompy/pull/59#discussion_r258555699. I'm happy to refactor both classes in a future PR

Reply:
Oh oops!! I guess I wasn't thinking as clearly back then. 🙈